reference/concepts/declarative_programming.md

```diff
@@ -0,0 +1,7 @@
+## Declarative programming
```

Comment:
There is an existing document describing declarative programming: https://github.com/exercism/v3/blob/master/reference/paradigms/declarative.md Could you remove this document from the PR?